code/modules/hivemind/wires.dm

```diff
@@ -0,0 +1,290 @@
+			var/icon/infected_icon = new('icons/obj/hivemind_machines.dmi', icon_state = "wires-[rand(1, 3)]")
+			var/icon/new_icon = new(subject.icon, icon_state = subject.icon_state, dir = subject.dir)
+			new_icon.Blend(infected_icon, ICON_OVERLAY)
+			new_machine.icon = new_icon
+			var/prefix = pick("strange", "interesting", "marvelous", "unusual")
+			new_machine.name = "[prefix] [subject.name]"
+		else
+			//of course, here we have a very little chance to spawn him, our mini-boss
+			if(prob(1))
+				new /mob/living/simple_animal/hostile/hivemind/mechiver(loc)
+				qdel(subject)
+				return
+			else
+				var/picked_machine
+				var/list/possible_machines = subtypesof(/obj/structure/hivemind_machine)
+				if(hive_mind_ai.hives.len < 10)
+					if(hive_mind_ai.evo_points < (hive_mind_ai.hives.len * 100)) //one hive per 100 EP
+						possible_machines -= /obj/structure/hivemind_machine/node
+					else
+						//we make new nodes asap, cause it has higher priority to survive, so we force it here
+						picked_machine = /obj/structure/hivemind_machine/node
+
+				//here we compare hivemind's EP with machine's required value
+				for(var/machine_path in possible_machines)
+					if(hive_mind_ai.evo_points <= hive_mind_ai.EP_price_list[machine_path])
+						possible_machines.Remove(machine_path)
+
+				if(!picked_machine)
+					picked_machine = pick(possible_machines)
+				var/obj/structure/hivemind_machine/new_machine = new picked_machine(loc)
+				new_machine.update_icon()
+
+	if(istype(subject, /mob/living) && !istype(subject, /mob/living/simple_animal/hostile/hivemind))
+		//human bodies
+		if(istype(subject, /mob/living/carbon/human))
+			var/mob/living/L = subject
+			for(var/obj/item/W in L)
+				L.drop_from_inventory(W)
+			var/M = pick(/mob/living/simple_animal/hostile/hivemind/himan, /mob/living/simple_animal/hostile/hivemind/phaser)
+			new M(loc)
+		//robot corpses
+		else if(istype(subject, /mob/living/silicon))
+			new /mob/living/simple_animal/hostile/hivemind/hiborg(loc)
+		//other dead bodies
+		else
+			var/mob/living/simple_animal/hostile/hivemind/resurrected/transformed_mob =  new(loc)
+			transformed_mob.absorb_the(subject)
+
+	qdel(subject)
+
+
+//////////////////////////////////////////////////////////////////
+/////////////////////////>RESPONSE CODE<//////////////////////////
+//////////////////////////////////////////////////////////////////
+
+
+//in fact, this is some kind of reinforced wires, so we can't take samples from it and inject something too
+//but we still can slice it with something sharp
+/obj/effect/plant/hivemind/attackby(obj/item/weapon/W, mob/user)
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+
+	var/weapon_type
+	if (W.has_quality(QUALITY_CUTTING))
+		weapon_type = QUALITY_CUTTING
+	else if (W.has_quality(QUALITY_WELDING))
+		weapon_type = QUALITY_WELDING
+
+	if(weapon_type)
+		if(W.use_tool(user, src, WORKTIME_FAST*0.30, weapon_type, FAILCHANCE_VERY_EASY, required_stat = STAT_ROB))
```

Comment:
make sure the failchance and worktime on these are harder and slower than cutting vines or mushrooms. this looks too low